### PR TITLE
[zh] sync concepts/security/security-checklist.md

### PR DESCRIPTION
**Pull request by KKtheGhost**
## PR Summary

- **EN upstream**: `content/en/docs/concepts/security/security-checklist.md`
- **Sync to**: `content/zh-cn/docs/concepts/security/security-checklist.md`

Sync check by`scripts/lsync.sh` on `main` branch:
```diff
$ scripts/lsync.sh content/zh-cn/docs/concepts/security/security-checklist.md
diff --git a/content/en/docs/concepts/security/security-checklist.md b/content/en/docs/concepts/security/security-checklist.md
index 99ad8b6180..ce96812567 100644
--- a/content/en/docs/concepts/security/security-checklist.md
+++ b/content/en/docs/concepts/security/security-checklist.md
@@ -97,6 +97,7 @@ For restricted LoadBalancer and ExternalIPs use, see
 [CVE-2020-8554: Man in the middle using LoadBalancer or ExternalIPs](https://github.com/kubernetes/kubernetes/issues/97076)
 and the [DenyServiceExternalIPs admission controller](/docs/reference/access-authn-authz/admission-controllers/#denyserviceexternalips)
 for further information.
+
 ## Pod security

 - [ ] RBAC rights to `create`, `update`, `patch`, `delete` workloads is only granted if necessary.
@@ -153,23 +154,20 @@ Memory limit superior to request can expose the whole node to OOM issues.

 ### Enabling Seccomp

-Seccomp can improve the security of your workloads by reducing the Linux kernel
-syscall attack surface available inside containers. The seccomp filter mode
-leverages BPF to create an allow or deny list of specific syscalls, named
-profiles. Those seccomp profiles can be enabled on individual workloads,
-[a security tutorial is available](/docs/tutorials/security/seccomp/). In
-addition, the [Kubernetes Security Profiles Operator](https://github.com/kubernetes-sigs/security-profiles-operator)
-is a project to facilitate the management and use of seccomp in clusters.
-
-For historical context, please note that Docker has been using
-[a default seccomp profile](https://docs.docker.com/engine/security/seccomp/)
-to only allow a restricted set of syscalls since 2016 from
-[Docker Engine 1.10](https://www.docker.com/blog/docker-engine-1-10-security/),
-but Kubernetes is still not confining workloads by default. The default seccomp
-profile can be found [in containerd](https://github.com/containerd/containerd/blob/main/contrib/seccomp/seccomp_default.go)
-as well. Fortunately, [Seccomp Default](/blog/2021/08/25/seccomp-default/), a
-new alpha feature to use a default seccomp profile for all workloads can now be
-enabled and tested.
+Seccomp stands for secure computing mode and has been a feature of the Linux kernel since version 2.6.12.
+It can be used to sandbox the privileges of a process, restricting the calls it is able to make
+from userspace into the kernel. Kubernetes lets you automatically apply seccomp profiles loaded onto
+a node to your Pods and containers.
+
+Seccomp can improve the security of your workloads by reducing the Linux kernel syscall attack
+surface available inside containers. The seccomp filter mode leverages BPF to create an allow or
+deny list of specific syscalls, named profiles.
+
+Since Kubernetes 1.27, you can enable the use of `RuntimeDefault` as the default seccomp profile
+for all workloads. A [security tutorial](/docs/tutorials/security/seccomp/) is available on this
+topic. In addition, the
+[Kubernetes Security Profiles Operator](https://github.com/kubernetes-sigs/security-profiles-operator)
+is a project that facilitates the management and use of seccomp in clusters.

 {{< note >}}
 Seccomp is only available on Linux nodes.
```
Sync check by`scripts/lsync.sh` on `sync/security-checklist` branch:
```diff
$ scripts/lsync.sh content/zh-cn/docs/concepts/security/security-checklist.md
content/zh-cn/docs/concepts/security/security-checklist.md is still in sync
```

Thanks to the reviewers in advance.
Best Regards,

Kivinsae Fang

### CODE REVIEW

File: content/zh-cn/docs/concepts/security/security-checklist.md [hunk 1]
```diff
@@ -281,42 +281,35 @@ Memory limit superior to request can expose the whole node to OOM issues.
...
+Since Kubernetes 1.27, you can enable the use of `RuntimeDefault` as the default seccomp profile
+for all workloads. A [security tutorial](/docs/tutorials/security/seccomp/) is available on this
+topic. In addition, the
+[Kubernetes Security Profiles Operator](https://github.com/kubernetes-sigs/security-profiles-operator)
+is a project that facilitates the management and use of seccomp in clusters.
 -->
 ### 启用 Seccomp {#enabling-seccomp}
 
-<!-- 按照英文原文翻译比较啰嗦，本小段是英文原文结合 Seccomp 简洁翻译的 -->
+Seccomp 代表安全计算模式（Secure computing mode），这是一个在 Linux 内核版本 2.6.12 被加入的特性。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/security/security-checklist.md:301]:
```suggestion
Seccomp 代表安全计算模式（Secure computing mode），这是一个自 Linux 内核版本 2.6.12 被加入的特性。
```

File: content/zh-cn/docs/concepts/security/security-checklist.md [hunk 2]
```diff
@@ -281,42 +281,35 @@ Memory limit superior to request can expose the whole node to OOM issues.
...
+它可以将进程的特权沙箱化，来限制从用户空间发起的对内核的调用。
+Kubernetes 允许你将加载到节点上的 Seccomp 配置文件自动应用于你的 Pod 和容器。
+
 Seccomp 通过减少容器内对 Linux 内核的系统调用（System Call）以缩小攻击面，从而提高工作负载的安全性。
-Seccomp 过滤器模式借助 BPF 创建了配置文件（Profile），文件中设置对具体系统调用的允许或拒绝，
-可以针对单个工作负载上启用这类 Seccomp 配置文件。你可以阅读相应的[安全教程](/zh-cn/docs/tutorials/security/seccomp/)。
+Seccomp 过滤器模式借助 BPF 创建了配置文件（Profile），文件中设置对具体系统调用的允许或拒绝。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/security/security-checklist.md:306]:
```suggestion
Seccomp 过滤器模式借助 BPF 创建具体系统调用的允许清单或拒绝清单，名为配置文件（Profile）。
```

**Comment by KKtheGhost** [comment]
Hi @windsonsea 

It was like the Netlify pipeline ran into some capacity issue now.
```
9:59:03 PM: Build exceeded maximum allowed runtime
```
If you have time could you please help re-trigger the pipeline one more time?
Thanks so much in advance.

Best Regards,

**Comment by windsonsea** [comment]
You need to trigger it again. I don't know why 😅

**Comment by KKtheGhost** [comment]
Copy that

I will do the re-trigger asap

**Comment by tanjunchen** [comment]
thanks!
/LGTM
/retest

**Comment by KKtheGhost** [comment]
Hi @tanjunchen 
I removed a unwanted space to re-trigger the build pipeline.
If you have time could you please re-LGTM for this pr? Thanks so much in advance.
```diff
- is a project that facilitates the management and use of seccomp in clusters. 
+ is a project that facilitates the management and use of seccomp in clusters.
```

**Pull request merged**